support-frontend/app/controllers/CreateSubscriptionController.scala

```diff
@@ -106,22 +112,26 @@ class CreateSubscriptionController(
       recaptchaService
         .verify(token, v2RecaptchaSecretKey)
         .leftMap(err => RequestValidationError(err))
-        .map(_.success)
         .subflatMap {
-          case true => Right(())
-          case false => Left(RequestValidationError("Recaptcha validation failed"))
+          case RecaptchaResponse(true, _) => Right(())
+
+          case RecaptchaResponse(false, _) =>
+            errorDetailedMessage("recaptcha failed")
+            Left(RequestValidationError("Recaptcha validation failed"))
```

Comment:
is this change just for clarity as I can't work out any functionality change?

Reply:
It was actually to allow us to access the `score` values of the `RecaptchaResponse`, which I think we're deemed we don't need. Happy to revert out.